### ISSUE TITLE
Investigate sphinx log-in authentication token refresh

### ISSUE THREAD

**Issue by ecurrencyhodler** [stakwork/sphinx-tribes#954]
**Describe the bug**
The logic for refreshing the authentication token is if a user is on any window with the base url: https://community.sphinx.chat. 

However occasionally I am forced to log-in even though I am active on one of the tabs.

One thing to potentially explore is if I am inactive on multiple tabs. 

**Desktop (please complete the following information):**
 - Browser: Chrome

**Comment by gouravmpk** [comment]
Does this Issue still persist ? @ecurrencyhodler

**Comment by ecurrencyhodler** [comment]
@gouravmpk yeah but it's hard to recreate. That's why it's backlogged for now. Happy to turn this into a bounty but we can only pay out if a fix is merged then tested.
